src/ds/logger.h

```diff
@@ -129,12 +132,21 @@ namespace logger
         j["m"].dump());
     }
 
-    void write(const std::string& log_line) override
+    virtual void write(const std::string& log_line) override
     {
       dump(log_line);
     }
   };
 
+  class JsonConsoleLogger : public JsonLogger
+  {
+  public:
+    void write(const std::string& log_line) override
+    {
+      std::cout << log_line << std::flush;
```

Comment:
Do we want to force a flush for every line? That's usually done for stderr but not stdout.